### PR TITLE
station change from RCH council

### PR DESCRIPTION
**Pull request by awdem**
https://app.asana.com/0/1205951895246773/1207213161209835/f

### CODE REVIEW

File: polling_stations/apps/data_importers/management/commands/import_rochdale.py [hunk 1]
```diff
@@ -63,4 +63,15 @@ def station_record_to_dict(self, record):
...
+
+        # Station change from council:
+        # old: Mobile Unit on the Car Park of Ogden Baptist Church, Cedar Lane, Newhey  OL16 4LD
+        # new: Mobile Unit on the Car Park of Elm Grove, OL16 4LH
+        if record.polling_place_id == "5422":
+            record = record._replace(
+                polling_place_address_1="Elm Grove",
+                polling_place_address_2="",
+                polling_place_postcode="OL16 4LH",
+            )
```
Comment (GeoWill) [inline polling_stations/apps/data_importers/management/commands/import_rochdale.py:75]: Does this replace all the parts of the old address that need replacing? Which columns contain the info are sometimes inconsistent between rows, so maybe safer to just replace all the `polling_place_address_n` fields.

**Review by GeoWill — COMMENTED** [review]
See inline comment

File: polling_stations/apps/data_importers/management/commands/import_rochdale.py (hunk 1 above)
Comment (awdem) [reply polling_stations/apps/data_importers/management/commands/import_rochdale.py:75]: Yes it should! I printed the whole record to compare it to the new one and then replaced or removed lines as needed.

Comment (awdem) [reply polling_stations/apps/data_importers/management/commands/import_rochdale.py:75]: I also checked it in qgis afterwards I should add.

**Review by GeoWill — APPROVED** [review]

**Pull request merged**